pallets/claims/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
-use codec::{Decode, Encode};
+use codec::{Decode, Encode, EncodeLike};
+use core::fmt::Debug;
```

Comment:
Is this needed?

Reply:
yes as t is currently implemented - we need specify debug and encodelike trait for the currency balance which we use.